MCprep_addon/util.py

```diff
@@ -228,7 +227,11 @@ def min_bv(version: Tuple, *, inclusive: bool = True) -> bool:
 
 
 def bv28() -> bool:
-	"""Check if blender 2.8, for layouts, UI, and properties. """
+	"""
+	Check if blender 2.8, for layouts, UI, and properties. 
+	
+	Deprecated in MCprep 3.5, but kept to avoid breakage for now...
+	"""
 	env.deprecation_warning()
 	return min_bv((2, 80))
```

Comment:
All of these references have been removed now. If not on this 3.6.0 branch, then check in dev (maybe merging dev into the milestone branch), in which case either way I think this function can just be removed.